### ISSUE TITLE
Error When Outputting Certain Field Types

### ISSUE THREAD

**Issue by super13** [dbcli/mycli#509]
select param from  p_task_status; 
will get a red 245. 
is it not support json type? is there any log or debug mode?

table:
CREATE TABLE `p_task_status` (
  `id` bigint(20) NOT NULL AUTO_INCREMENT,
  `task_id` varchar(255) DEFAULT NULL,
  `host` varchar(255) DEFAULT NULL,
  `status` int(10) DEFAULT NULL,
  `status_msg` varchar(255) DEFAULT NULL,
  `start_time` timestamp NULL DEFAULT CURRENT_TIMESTAMP ,
  `end_time` timestamp DEFAULT '0000-00-00 00:00:00',
  `time_spends` bigint(20) DEFAULT NULL,
  `param` json DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;

**Comment by tsroten** [comment]
@super13 Hello! I'm sorry that you're having trouble with your JSON values. Mycli does support JSON data, e.g.:
```
localhost:test_db> select param from p_task_status
+--------------------------------------+
| param                                |
+--------------------------------------+
| {"key1": "value1", "key2": "value2"} |
+--------------------------------------+
1 row in set
Time: 0.016s
```

To find out more information about your error, you can check the log file. By default it is `~/.mycli.log`. Let us know what you find and we'll be happy to help you debug it.

**Comment by tsroten** [comment]
Hi @super13! @fpietka has replicated the issue. It's a bug with mycli. We'll get it fixed as soon as we can. Thanks for reporting this!

**Comment by LiraNuna** [comment]
For posterity, this is in the logs:

```
2017-09-12 22:22:20,606 (20851/MainThread) mycli.main ERROR - sql: u'SELECT media.metadata FROM media LIMIT 1', error: KeyError(245,)
2017-09-12 22:22:20,606 (20851/MainThread) mycli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/lib/python2.7/dist-packages/mycli/main.py", line 541, in one_iteration\n    max_width)\n  File "/usr/local/lib/python2.7/dist-packages/mycli/main.py", line 850, in format_output\n    for col in cur.description]\nKeyError: 245\n'
```

**Comment by tsroten** [comment]
Noting that this happens with null values as well: https://github.com/dbcli/mycli/issues/515

**Comment by tsroten** [comment]
@super13 Good news! The fix for this was just released in version 1.13.0. If you installed via pip you can upgrade by running `pip install -U mycli`

It's also in Homebrew as well: `brew upgdate && brew upgrade mycli`.
